flink-connector-jdbc-core/src/main/java/org/apache/flink/connector/jdbc/internal/options/InternalJdbcConnectionOptions.java

```diff
@@ -179,7 +179,7 @@ public Builder setDBUrl(String dbURL) {
 
         /**
          * optional, Handle the SQL dialect of jdbc driver. If not set, it will be infer by {@link
-         * JdbcDialectLoader#load} from DB url.
+         * JdbcFactoryLoader#load} from DB url.
```

Comment:
```suggestion
         * JdbcFactoryLoader#load(String, ClassLoader)} from DB url.
```

Reply:
changed to JdbcFactoryLoader#loadDialect